lib/src/Pos/Logic/Pure.hs

```diff
@@ -0,0 +1,266 @@
+-- | A pure logic layer. As such, it does nothing, but won't crash.
+
+module Pos.Logic.Pure
+    ( pureLogic
+    , blockVersionData
+    ) where
+
+import           Universum
+
+import           Data.Default (def)
+import           Data.ByteString (ByteString)
+import qualified Data.ByteString as BS
+import           Cardano.Crypto.Wallet (xsignature)
```

Comment:
Oh, wait, no, direct crypto imports are forbidden. In the case of `blockSignature` you must use `unsafeHash` or something similar.

Reply:
Yeah I looked into this. I just want a signature. I don't want to actually sign anything, I just want a signature (i.e. the appropriate number of bytes). Can I get that somehow?

Reply:
Well, I don't see any reason why `unsafeHash ()` wouldn't do it for you. We're not fighting for performance here anyway. You can put `dummyHash :: Hash a` into `Pos.Crypto.Hashing` if you want to.

Reply:
I need an `XSignature` here, not a hash.

Reply:
OK I replaced it with `signRaw`, using the `SecretKey` that's generated along with the public key to give the leader key in the consensus data. But now I need a `Given ProtocolMagic` constraint D: I just used `ProtocolMagic 0`.